### ISSUE TITLE
Error while adding a link in a table cell

### ISSUE THREAD

**Issue by ddmng** [careteditor/caret#443]
Automatic error reporting triggered, no evidence of any error to me.

I was adding a link after `testcell3` string in the reported test table. Once added `[](./)` the auto-suggest started proposing files. Typed some character and deleted with `CTRL-Backspace` then the red "Error has occurred" appeared at the bottom.
 
Test table:
```
| testheader1 | testheader2 | Testheader3      |
| ----------- | ----------- | ---------------- |
| Testcell1   | testcell2   | testcell3  |
```

The following error has occurred:
```
{
    "message": "Cannot read property 'top' of undefined",
    "version": "3.2.2",
    "os": "Linux 4.8",
    "path": "web/Editor/Editor.Autocomplete.js",
    "code": "t.top+=Math.ceil(this._editor.dimensions.lineHeight/2),this.element.style.display=\"\",this.element.style.top=t.top+\"px\",this.element.style.left=t.left+\"px\",this._editor.command.get(\"autocomplete-close\").enabled=!0,this._editor.command.get(\"autocomplete-prev-item\").enabled=!0,this._editor.command.get(\"autocomplete-next-item\").enabled=!0,this._editor.command.get(\"autocomplete-select-item\").enabled=!0}.bind(this)))}},",
    "line": 5,
    "column": 2423,
    "stack": [
        "at t.<anonymous> (web/Editor/Editor.Autocomplete.js:5:2423)",
        "at Object.e [as callback] (web/Events.js:5:773)",
        "at t.value (web/Events.js:5:1330)",
        "at t.value (web/Editor/Editor.js:5:17933)"
    ]
}
```

**Comment by astoilkov** [comment]
This is now fixed in the latest Caret 3.2.3 release.

Closing this issue. Thanks for reporting.

**Comment by ddmng** [comment]
It works without any error now.
thanks
